recipe/meta.yaml

```diff
@@ -19,7 +19,7 @@ requirements:
     - pip
   run:
     - python >=3.6,<4.0
-    - julia >=1.5.0
+    - juliaup >=1.5.0
```

Comment:
Ah, this seems to not fix the Julia version, but the juliaup version... How should fixing the julia version be handled?

Reply:
You would need a julia package that uses juliaup. The easiest route would be an alias that does `julia +1.9` or something like that.